src/utils.js

```diff
@@ -192,6 +193,12 @@ export const dollarString = number => formatNumber(number, '$0,0')
  */
 export const integerString = number => formatNumber(number, '0,0')
 
+/**
+ * @param {number} a float
```

Comment:
Per JSDoc syntax, this needs to reference the name of the variable. So, this will need to be something like:

```
* @param {number} number A float
```

Reply:
oops, number/number! good catch